lib/resources/r4/related_person_patch.yaml

```diff
@@ -126,6 +126,45 @@ operations:
         <li>When specifying a `period`, the `start` and `end` fields must include a time component with a timezone.</li>
       </ul>
 
+  - name: add-telecom
+    path: /telecom/-
+    operation: add
+    type: ContactPoint
+    url: https://hl7.org/fhir/R4/related-person-definitions.html#RelatedPerson.telecom
+    description: Add a new contact detail for the related person.
+    example: |
+      [
+          {
+          	"op": "add",
+          	"path": "/telecom/-",
+          	"value": {
+                  "use": "home",
+                  "system": "email",
+                  "value": "test@test.com",
+                  "period": {
+                      "start": "2018-01-05T22:24:32.000Z"
+                  }
+              }
+          },
+          {
+          	"op": "add",
+          	"path": "/telecom/-",
+          	"value": {
+                  "use": "home",
+                  "system": "phone",
+                  "value": "913-444-8881",
+                  "period": {
+                      "start": "2018-01-05T22:24:32.000Z"
+                  }
+              }
+          }
+      ]
```

Comment:
nit: tabs should be 2 spaces.

Reply:
updated here: https://github.com/cerner/fhir.cerner.com/pull/559/commits/9a5355ede6a1e2988bb7e041faf5224a1ef4acdb#diff-a0c01bda612e4656e850590d65d36528f1caec8a70317c2ae4cb71d8095c9e7dR136-R159